packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -1,6 +1,9 @@
+%global dnf_install (0%{?rhel} > 7) || (0%{?fedora} > 26)
+%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
```

Comment:
@ekohl Nice catch on the '/'

In reality there is no python_sitelib macro in el5 so I found this workaround on the internet.

```
[root@2b6de9090795 ~]# python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"
/usr/lib/python2.4/site-packages
```

Reply:
Always pays off to question when something doesn't look right. Either you learn something you didn't know or it's a bug :)